packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -137,14 +137,20 @@ $btn-sizeXSmall-lineHeight      : 1;
         background-color: $btn-default-bgColor--hover;
     }
 
-    &:focus {
-        outline: 2px dotted $btn-default-outline-color;
+    // Hide focus styles if they're not needed, for example, when an element receives focus via the mouse.
+    &:focus:not(:focus-visible) {
+        outline: 0;
+    }
+
+    // Show focus styles on keyboard focus.
+    &:focus-visible {
+        outline: 2px solid $btn-default-outline-color;
```

Comment:
I thought that this dotted outline came from new pie design

Reply:
Have just checked and, yes. it looks like buttons should have dotted outlines. They should also have a rounded outline but I've not been able to figure out how to do that with `outline` as there is no outline radius property. 

I'll set this back to dotted outlines and will have a think about the rounded styles.